Exiled.API/Features/Map.cs

```diff
@@ -396,6 +396,17 @@ public static void ShowHint(string message, float duration)
         /// </summary>
         public static void ClearBroadcasts() => Server.Broadcast.RpcClearElements();
 
+        /// <summary>
+        /// Gets a random spawn point of a <see cref="RoleType"/>.
+        /// </summary>
+        /// <param name="roleType">The <see cref="RoleType"/> to get the spawn point from.</param>
+        /// <returns>Returns the spawn point <see cref="Vector3"/>.</returns>
+        [Obsolete("Moved to Exiled.API.Extensions.Role.")]
```

Comment:
Complete the message.

Reply:
I wasn't entirely sure what you meant, but I hope what I added was what you meant because I need to go to sleep, if it isn't, I'll fix it in the morning.